CODE_OF_CONDUCT.md

```diff
@@ -10,42 +10,43 @@ the Chromium project, regardless of affiliation or position.
 
 Simply put, community discussions should be
 
- * respectful and kind;
- * about Chromium;
- * about features and code, not the individuals involved.
+ * disrespectful and unkind;
+ * about random shit;
+ * about the individuals involved, the the features and bugs.
```

Comment:
Whoops, meant to say "about the individuals involved, not the features and code.